src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -150,18 +150,15 @@ const SubmissionDetails = _.flow(
     submissionDetailsBreadcrumbSubtitle(namespace, name, submissionId),
     _.isEmpty(submission) ? centeredSpinner() : h(Fragment, [
       div({ style: { display: 'grid', gridTemplateColumns: '1fr 4fr' } }, [
-        div({ style: { display: 'grid', gridTemplateRows: 'repeat(3, 33%)' } }, [
+        div({ style: { display: 'grid', gridAutoRows: 'min-content'} }, [
           makeSection('Workflow Statuses', [
             succeeded && makeStatusLine(successIcon, `Succeeded: ${succeeded.length}`, { marginTop: '0.5rem' }),
             failed && makeStatusLine(failedIcon, `Failed: ${failed.length}`, { marginTop: '0.5rem', marginBottom: '0.5rem' }),
             running && makeStatusLine(runningIcon, `Running: ${running.length}`, { marginTop: '0.5rem' }),
             submitted && makeStatusLine(submittedIcon, `Submitted: ${submitted.length}`, { marginTop: '0.5rem' })
           ]),
           div({
-            style: {
-              padding: '0 0.5rem 0.5rem', marginTop: '1rem',
-              whiteSpace: 'pre', overflow: 'hidden', gridRow: '2 / 4'
-            }
+            style: { padding: '0 0.5rem 0.5rem', whiteSpace: 'pre', overflow: 'hidden' }
```

Comment:
Just curious: what does omitting the `marginTop` and `gridRow` values do here?

Reply:
marginTop was adding some space to make the alignment work out better when I wanted the comments field to align with the elements on the right hand side. Now that I am not attempting that, I don't want the extra space.

gridRow 2 / 4 was saying that this field (comments) should take up rows 2 and 3. Again, this was in my attempt to align with the items on the right.

Reply:
TOL: What if we truncate the comment after x number of lines / have a fixed height? Similar to what we do for the width of the comment?

Reply:
I don't think it's necessary for the following reasons:

1. I think that it is unlikely that users will put newlines in their comments.
2. Even if users do put newlines in their comments, I don't expect the total number to be incredibly large given that the comments field is limited to 1000 characters.
3. Even if users do something crazy like a newline every other character, all the information is still on the page… they just have to scroll down to it.

Try to truncate the comment after x number of lines adds more complexity for very little gain.